app/models/bike.rb

```diff
@@ -69,7 +69,7 @@ class Bike < ActiveRecord::Base
   enum propulsion_type: PropulsionType::SLUGS
 
   default_scope {
-    includes(:tertiary_frame_color, :secondary_frame_color, :primary_frame_color, :current_stolen_record)
+    includes(:tertiary_frame_color, :secondary_frame_color, :primary_frame_color, current_stolen_record: [:state, :country])
```

Comment:
I realize we've already committed to this default scope thing, but adding more to this still scares me.

Probably don't do anything about this. I guess we can just make the default scope everything we want, and just call unscoped on `Bike` wherever that makes sense. I just worry about it.

Reply:
I'm not crazy about it either, actually — I'll kill it. There's an N+1 query somewhere making Bullet mad, but adding to the default scope is overkill. Shouldn't be hard to track down.

Reply:
I think it would be great to add a named scope and put the includes in there. And then use that named scope all over.

I think the default scope hiding bikes that need to be hidden makes sense (even though I'm not in love with it).

But even moving the listing order into the named searching scope probably makes sense (or making it its own named scope and including it in the named searching scope, etc).

Reply:
Updated in fe316716